README.md

```diff
@@ -1 +1,55 @@
-# final-portofolio
\ No newline at end of file
+
+# Finish Mobile Portofolio
+This project is The Mobile Portofolio Website.In this project I added the linter , html and css file.The main purpose of this project is to setting up your Protofolio Website.
+
+
+## Built With
+
+- Major languages used html and css
+- Frameworks used visual studio code
+- Technologies used Laptop
+   
```

Comment:
[OPTIONAL] Please note that by [frameworks](https://hackr.io/blog/what-is-frameworks) it means ruby or react frameworks. Visual studio code is an [IDE](https://www.redhat.com/en/topics/middleware/what-is-ide).

You can maybe replace laptop with vscode and remove the laptop. Also i suggest you delete the frameworks since you haven't learnt any yet